techmatters/webchat
# Mobile Optimization

Primary reviewer: @mmythily 

## Description
This PR optimizes webchat for mobile. When the widget is open on a mobile device, it will take the full screen.


### Related Issues
Fixes https://tech-matters.atlassian.net/browse/CHI-1355

### Verification steps
- Open the widget on desktop: nothing should be changed
- Open the widget on a mobile device: it should take full screen
